### PR TITLE
Updating the package to a new version

### PR DESCRIPTION
**Pull request by Vergara92**
Upgrading the package version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "to-do-list",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (Nuzukaru) [inline package.json:3]: why are you cahging that

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
   "description": "",
   "main": "index.js",
   "scripts": {
```
Comment (Nuzukaru) [inline package.json:6]: Why?

**Review by Nuzukaru — APPROVED** [review]
Seens gud

File: package.json (hunk 2 above)
Comment (Vergara92) [reply package.json:6]: 💯

**Pull request merged**